### PR TITLE
[PTT-187] configure service empty installments dates

### PR DESCRIPTION
**Pull request by filma80**
#### List of Changes
Empty dates are no longer valid in configure service page.

#### Motivation and Context
Empty dates are not allowed for configure service.

#### How Has This Been Tested?
Unit and integration tests.

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/app/pages/tributi-page/tributi-step2-definisci-rate/tributi-step2-definisci-rate.component.ts [hunk 1]
```diff
@@ -193,7 +193,7 @@ export class TributiStep2DefinisciRateComponent implements OnInit {
 
   private areInstallmentsDatesValid(): boolean {
     for (const elem of this.f.installments.value) {
-      if (!this.isDateAfter_30_06_2021(elem.dueDate) && elem.percentageSecondary > 0) {
+      if (!elem.dueDate || (!this.isDateAfter_30_06_2021(elem.dueDate) && elem.percentageSecondary > 0)) {
```
Comment (GiovanniMancini) [inline src/app/pages/tributi-page/tributi-step2-definisci-rate/tributi-step2-definisci-rate.component.ts:196]: This data may change. It shouldn't be static

**Review by GiovanniMancini — CHANGES_REQUESTED** [review]
30-06-2021 shoudln't be harcoded

**Comment by filma80** [comment]
It will be added a new boolean to environment constants to enable 30/06 checks.

**Comment by filma80** [comment]
Added boolean to enable or disable checks.

**Review by GiovanniMancini — APPROVED** [review]

**Comment by GiovanniMancini** [comment]
@filma80 please resolve conflicts with the main branch

**Pull request merged**